atomic_red_team/labels.py

```diff
@@ -152,12 +152,13 @@ def save_labels_and_maintainers(self, pr):
                 labels.append(self.labels[p])
             if p in self.maintainers:
                 maintainers += self.maintainers[p]
+
+        pull_request = self.repo.get_pull(pr)
+        pull_request.set_labels(*labels)
+        pull_request.add_to_assignees(*maintainers)
+
```

Comment:
`assign-labels` action has been removed. Adding Github SDK to support the same feature